test.py

```diff
@@ -8,4 +8,5 @@
 
 #For now this is just printing off all of the 
 for square in board.squares:
-	print(board.squares[square].piece_moves)
\ No newline at end of file
+	print(board.squares[square].piece_moves)
+	
```

Comment:
- The comment in line 9 seems to be an incomplete thought
- If you are running this file and have it to where you are happy with the moves, consider making this a proper test with an assertion so that you can tell if the answer changes and catch future breakages.

Reply:
Before merging this PR, I'd suggest refactoring so that the board-drawing class returns a string rather than prints to the screen, and writing a unit test showing that when the board is first set up before any moves, the drawing representation is what is expected. That will give us some confidence as more things get built that we haven't broken what was already working.

Reply:
This should hav ebeen in my .gitignore apologies.

Reply:
I don't think so! There is nothing about the file name `test.py` that should match something in your gitignore file. Instead, I'd just expect you to not add this file to a commit until it's something you are ready to have in the public repo. 

Are you currently manually adding files to each commit? Are you aware of ["interactive staging"](https://git-scm.com/book/en/v2/Git-Tools-Interactive-Staging) so you can go through and pick which changes in your working directory you want to add to the next commit?